stardog-spring-batch/build.gradle

```diff
@@ -36,9 +36,9 @@ dependencies {
 	compile ('org.springframework.batch:spring-batch-core:3.0.5.RELEASE') 
 	compile ('org.springframework.batch:spring-batch-infrastructure:3.0.5.RELEASE') 
 
-	compile ('com.complexible.stardog:client-http:7.2.1')
-	compile ('com.stardog:stardog-spring:7.2.1')
-	testCompile ('com.complexible.stardog:server:7.2.1') {
+	compile ('com.complexible.stardog:client-http:7.3.2')
+	compile ('com.stardog.ext:stardog-spring:1.0.0')
+	testCompile ('com.complexible.stardog:server:7.3.2') {
```

Comment:
We can/should keep this as a variable in the base build.gradle so we only have to edit it in one place. The docs could potentially pull from that place too...